tests/integration/models/setup_utils.py

```diff
@@ -371,7 +314,7 @@ def time_dependent_bc_values_mechanics(
         return values.ravel("F")
 
 
-class NoPhysics(pp.ModelGeometry, pp.SolutionStrategy, pp.DataSavingMixin):
+class NoPhysics(pp.ModelGeometry, pp.SolutionStrategy, pp.DataSavingMixin, pp.BoundaryConditionMixin):
```

Comment:
Please run black

Reply:
Running black on tests/ modify 16 files. Should I commit them?

Reply:
Yes, please

Reply:
Done